pulsar-broker/src/test/resources/log4j.properties

```diff
@@ -1,41 +0,0 @@
-#
-# Licensed to the Apache Software Foundation (ASF) under one
-# or more contributor license agreements.  See the NOTICE file
-# distributed with this work for additional information
-# regarding copyright ownership.  The ASF licenses this file
-# to you under the Apache License, Version 2.0 (the
-# "License"); you may not use this file except in compliance
-# with the License.  You may obtain a copy of the License at
-#
-#   http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing,
-# software distributed under the License is distributed on an
-# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-# KIND, either express or implied.  See the License for the
-# specific language governing permissions and limitations
-# under the License.
-#
-
-
-#
-# Pulsar-Broker Tests Logging Configuration
-#
-# Format is "<default threshold> (, <appender>)+
-
-log4j.rootLogger=OFF, CONSOLE
```

Comment:
I couldn't find a way to have a single file to control all the modules. 

With this patch the level can be controlled with system properties. Take a look at the top level `pom.xml` where in the surefire config we can have: 

```
 -Dorg.slf4j.simpleLogger.log.org.apache.bookkeeper.mledger=info
```